index.js

```diff
@@ -63,7 +63,6 @@ class DHT extends EventEmitter {
 
     this.table.on('row', this._onrow)
 
-    if (this.ephemeral === false) this.io.ephemeral = false
```

Comment:
It was added here: https://github.com/mafintosh/dht-rpc/commit/bfc0e19e920261eb2047563e9d54f101e120304b

I remember I triple checked that it should be ok to remove it, but maybe you added it for other reason?